com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -0,0 +1,94 @@
+{
+  "app-id": "com.jetbrains.IntelliJ-IDEA-Community",
+  "command": "idea",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "18.08",
+  "separate-locales": false,
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--device=all",
+    "--device=dri",
+    "--env=IDEA_JDK=/app/extra/idea-IC/jre64/",
+    "--env=IDEA_VM_OPTIONS=/app/extra/idea-IC/bin/idea64.vmoptions",
+    "--filesystem=host",
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [
+    {
+      "name": "libsecret",
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/*.la",
+        "/lib/pkgconfig"
+      ],
+      "config-opts": [
+        "--disable-gtk-doc",
+        "--disable-introspection",
+        "--disable-manpages",
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "f4fa198d4e131f94d9ec84cfeaf6c124245e7ff3c0af11c76cc35332bfae5585",
+          "url": "https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.18.6/libsecret-0.18.6.tar.gz"
+        }
+      ]
+    },
+    {
+      "name": "idea-ic",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 idea.sh /app/bin/idea",
+        "install -Dm0755 -t /app/bin/ apply_extra",
+        "install -Dm0644 -t /app/share/applications/ com.jetbrains.IntelliJ-IDEA-Community.desktop",
+        "install -Dm0644 -t /app/share/icons/hicolor/scalable/apps/ com.jetbrains.IntelliJ-IDEA-Community.svg",
+        "install -Dm0644 -t /app/share/metainfo/ com.jetbrains.IntelliJ-IDEA-Community.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "extra-data",
+          "filename": "ideaIC.tar.gz",
+          "only-arches": ["x86_64"],
+          "sha256": "387a4a55eee6aaac3a82f392c8d675294987b51ce8e91788b04e1218565424fe",
+          "size": 541751086,
+          "url": "https://download.jetbrains.com/idea/ideaIC-2018.3.1.tar.gz"
```

Comment:
Isn't it free and under free license like PyCharm is?

Reply:
Yes, it's free; this one is governed by `Apache-2.0`. I think I know what you meant: do not use `extra-data`, right?

Reply:
Exactly.